SecretSalty/src/main/java/com/secretsalty/secretsalty/repository/ParticipantRepository.java

```diff
@@ -0,0 +1,90 @@
+package com.secretsalty.secretsalty.repository;
+
+import com.secretsalty.secretsalty.repository.model.Participant;
+import data.jooq.tables.AppParticipant;
+import data.jooq.tables.records.AppParticipantRecord;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.jooq.DSLContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@Repository
+public class ParticipantRepository {
+
+  private final DSLContext dslContext;
+
+  @Autowired
+  public ParticipantRepository(DSLContext dslContext) {
+    this.dslContext = dslContext;
+  }
+
+  @Transactional
+  public Participant create(Participant newParticipant) {
+    LocalDateTime now = LocalDateTime.now();
+    newParticipant.setCreated(now);
+    newParticipant.setLastModified(now);
+
+    return this.dslContext
+        .insertInto(AppParticipant.APP_PARTICIPANT)
+        .set(this.toRecord(newParticipant))
+        .returning()
+        .fetchOneInto(Participant.class);
+  }
+
+  @Transactional
+  public List<Participant> readAllForUser(UUID userUuid) {
```

Comment:
I'd maybe change the name here; readAllForUser is a little bit confusing?

readAllByUser maybe? 🤔